### ISSUE TITLE
Rider 50 not supported

### ISSUE THREAD

**Issue by katterfelto** [Pitmairen/bryton-gps-linux#16]
The rider 50 seems similar to the rider 20p the differences that I can see are:

The contents for /dev/disk/by-id are:
usb-Bryton_Flash_Disk_PRAHA_Device-0:0@
usb-Bryton_Flash_Disk_PRAHA_Device-0:0-part1@
usb-Bryton_Flash_Disk_PRAHA_Device-0:0-part5@
usb-Bryton_MicroSD_PRAHA_Device-0:1@

The files are in a different directory structure. The cycling data is stored in the following path below the mount point:

thalia/applications/CYCLING/DATA/TRACELOG

This folder seems to have all the data for each run. Each run contains 4 files, the first part of the name is numeric and unique to that recording and the second half finish as follows: *.-GPS.dat *-STA.xml *-TRN.dat *-USR.xml

If I can figure out how I will attach an archive with a set of sample files.

I've emailed you the 4 files, but read a bit more of the 20p issue thread. So the following link has the 4 data files already sent to you plus the matching exported bdx file.

http://mail.sim-soft.co.uk/~dm/Rider50data.zip

**Comment by piggz** [comment]
@DaisyDave sorry to hi-jack this thread....ive just got a rider 50 and am a linux user/dev ... but, on my 50, the maps dont load, with errors about the map engine being too old .... if its no trouble, are you able to zip up your sd card and upload it somewhere so that I might get this working properly.  mine is the navteq version and there seems no way to restore the sd card data :/

**Comment by Pitmairen** [comment]
I don't have a rider 50, but you probably have to update your device software to the latest update from Bryton. To do this you of course have to have a windows or mac machine. You can also use a virtual machine to do this.

There is also a Device recovery tool you can download from this page: 
http://corp.brytonsport.com/products/support_sdownload?lang=en

**Comment by piggz** [comment]
yeah, i tried all that already, cheers..did fix last night though, renamed mapengine.dll and version.ini, which forced bb2 to update the files. new files were larger and higher version, seems to work fine now! :)

**Comment by fu7zed** [comment]
Hi everyone,

Sorry, my message is not related to the issue, but I would just need the help of another user of this device.

I updated my Bryton rider 50 with Bryton Bridge 2 and it doesn't work since then (map engine is outdated).

Visibly, seeing piggz issue, I need the latest mapengine.dll and version.ini file (at the root of the SD card).

I tried to empty (and backup) the contents of the SD card, use the device recovery tool, updating again, etc, etc. Visibly it cannot bring mapengine.dll and version.ini back

I've contacted Bryton support many times, they did not respond.

Can someone having these 2 files PM me on facebook or contact me at aubreycoupart@gmail.com ? Any help would be truly appreciated, 3 weeks of struggling to make this shit work, I start to go mad. This is the period of the year when I train everyday and not being able to use GPS is so frustrating :(((

Many many many thanks for reading.

**Comment by LisetteZB** [comment]
@piggz I know its a long time ago but I ran into the same engine is too old problem when starting to use my old bryton 50 again and cant find a fix. Can I ask how you renamed these files so they were forced to update, if you can still remember...
